src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
```

Comment:
Using `String(robot[key])` is unnecessary since object keys are always strings. If the value is not a string, it will be coerced to a string automatically when used as a property key.